### PR TITLE
fix: add Santosh1176

### LINKED ISSUE #666: Anas Khan
Screenshot (28)

### PR DESCRIPTION
**Pull request by Santosh1176**
resolve #1130

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Santosh1176.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91916466/169870632-0ab75c94-7045-4aee-be36-6ecd734d06b4.png",
+  "issueId": 666,
```
Comment (Panquesito7) [inline profiles/Santosh1176.json:3]: I think this is the wrong issue number? 👀

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Santosh1176.json (hunk 1 above)
Comment (Santosh1176) [reply profiles/Santosh1176.json]: I’ve made necessary changes to #issue number. I hope it’s ok now.

File: profiles/Santosh1176.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91916466/169870632-0ab75c94-7045-4aee-be36-6ecd734d06b4.png",
+  "issueId": 1131,
```
Comment (schmelto) [inline profiles/Santosh1176.json:3]:
```suggestion
  "issueId": 1130,
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Look good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]

**Comment by Panquesito7** [comment]
I think #1130 is also the wrong ID? 👀 Looks like it's a PR.

**Pull request merged**